keanu-project/src/main/java/io/improbable/keanu/vertices/intgr/nonprobabilistic/operators/binary/IntegerAdditionVertex.java

```diff
@@ -1,17 +1,19 @@
 package io.improbable.keanu.vertices.intgr.nonprobabilistic.operators.binary;
 
 import io.improbable.keanu.tensor.intgr.IntegerTensor;
+import io.improbable.keanu.vertices.LoadParentVertex;
+import io.improbable.keanu.vertices.SaveableVertex;
 import io.improbable.keanu.vertices.intgr.IntegerVertex;
 
-public class IntegerAdditionVertex extends IntegerBinaryOpVertex {
+public class IntegerAdditionVertex extends IntegerBinaryOpVertex implements SaveableVertex {
 
     /**
      * Adds one vertex to another
      *
      * @param a a vertex to add
      * @param b a vertex to add
      */
-    public IntegerAdditionVertex(IntegerVertex a, IntegerVertex b) {
+    public IntegerAdditionVertex(@LoadParentVertex(A_NAME) IntegerVertex a, @LoadParentVertex(B_NAME) IntegerVertex b) {
```

Comment:
How come we have `a` and `b` for integer binary ops and `left` and `right` for double binary ops? Seems like a hang over from the olden days. Can we make these consistent while we're down there?

Reply:
Done